uclouvain-faculte-de-droit.csl

```diff
@@ -0,0 +1,558 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" default-locale="fr-BE" and="text" et-al-min="5" et-al-use-first="1" initialize-with="." page-range-format="expanded" demote-non-dropping-particle="never" xmlns="http://purl.org/net/xbiblio/csl">
```

Comment:
`default-locale="fr-BE"`

@adam3smith, we should probably change this to "fr-FR", since we don't have a "fr-BE" CSL locale. (and add the language to the style title)